integration_test/README.md

```diff
@@ -296,3 +296,22 @@ https://storage.googleapis.com/ops-agents-public-buckets-vendored-deps/mirrored-
 
 instead of the original URL.
 
+# Soak Test
+
+This program launches a VM with a load generator running on it to simulate
+load. To run it, replace `my_project` with your project (and adjust any other
+parameters as desired) and run:
+
+```
+PROJECT=my_project \
+  DISTRO=debian-11 \
+  ZONES=us-central1-b \
+  TTL=100m \
+  LOG_SIZE_IN_BYTES=1000 \
+  LOG_RATE=1000 \
+  go run -tags=integration_test soak_test/launcher.go
```

Comment:
You should never use `go run` with a filename or filenames.

Put the launcher in its own directory and use

```
go run ./soak_test/cmd/launcher
```

(Note the leading `./` is required, making it a relative package path.)

Reply:
> You should never use go run with a filename or filenames.

why?

also, because of the way embed works, I'd need to put log_generator.py inside cmd too, which seems a bit odd. But I'll do it if you have some reason for it.

Reply:
`go run` will build the filenames in a subtly different way that can cause hard-to-debug problems. In particular it will compose a new main package from the passed filenames, which is completely different from the normal one-package-per-directory rule (that all other tools, like `go test`, assume). So the usual advice is simply "do not use go run with filename(s)". By convention, `main` packages should be defined in directories named `.../cmd/foo`.